### DISCUSSION TITLE
Any way to deduplicate module configs between monitors?

### DISCUSSION THREAD

**Discussion by rbutoi** [Alexays/Waybar#1840] [Q&A]
I have [multi-output config](https://github.com/Alexays/Waybar/wiki/Configuration#configuration-of-multiple-outputs) for which the only differences between outputs is the _choice_ of modules. The config of the modules is the same, but I'm currently forced to copy that, which adds some toil whenever I want to make a change:

```
[{
  "output": "laptop",
  "modules-left": [ ... ],
  "modules-right": [ ... ],
  "module1": { a config },
  "module2": { a config },
  ...
}, {
  "output": "external1",
  "modules-left": [ ...something else... ],
  "modules-right": [ ...something else... ],
  "module1": { the same config copied },
  "module2": { ...etc... },
  ...
}, {
  "output": "external2",
  ...
]
```

Is there a way around this?

**Comment by topaxi** [comment]
I'm facing the same issue. As I can't find anything, I decided on having a tiny script and my own data structure cobbling together the final duplicated output.
